packages/dd-trace/src/runtime_metrics/runtime_metrics.js

```diff
@@ -160,12 +185,41 @@ function captureMemoryUsage () {
   client.gauge('runtime.node.mem.rss', stats.rss)
   client.gauge('runtime.node.mem.total', os.totalmem())
   client.gauge('runtime.node.mem.free', os.freemem())
+  client.gauge('runtime.node.mem.external', stats.external)
+  // TODO: Add arrayBuffers to the metrics. That also requires the
+  // node_metadata.csv to be updated for the website.
+  //
+  // client.gauge('runtime.node.mem.arrayBuffers', stats.arrayBuffers)
+}
 
-  stats.external && client.gauge('runtime.node.mem.external', stats.external)
+function captureUptime () {
+  // WARNING: lastTime must be updated in the same interval before this function is called!
+  // This is a faster `process.uptime()`.
+  client.gauge('runtime.node.process.uptime', Number((lastTime - startTime) / 1_000_000_000n))
```

Comment:
Again, with `perf_hooks.performance.now()`, `lastTime` is effectively uptime, so you could do this:
```
client.gauge('runtime.node.process.uptime', lastTime / 1_000)
```
or
```
client.gauge('runtime.node.process.uptime', (lastTime - 500) / 1_000)
```
if you wanted that 500ms offset.

Reply:
I added the offset, while I am not 100% certain which is now correct. It is difficult to tell due to the mocking. I might check again with the former implementation if all values align.